.github/workflows/remoteconfig.yml

```diff
@@ -0,0 +1,39 @@
+name: remoteconfig
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'FirebaseRemoteConfig**'
+    - '.github/workflows/remoteconfig.yml'
+    - 'Gemfile'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+
+
+  quickstart:
+    env:
+      plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+      signin_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set xcode version
+      run: sudo xcode-select -switch /Applications/Xcode_11.3.app
+    - name: Setup quickstart
+      run: scripts/setup_quickstart.sh config
+    - name: Install Secret GoogleService-Info.plist
+      run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/qs-auth.plist.gpg \
+          quickstart-ios/config/GoogleService-Info.plist "$plist_secret"
+    - name: Install Secret FIREGSignInInfo.h
+      run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/FIREGSignInInfo.h.gpg \
+          quickstart-ios/TestUtils/FIREGSignInInfo.h "$signin_secret"
+    - name: Test quickstart
+      run: |
+        scripts/test_quickstart.sh Config
```

Comment:
Style nit: This can fit on one line so the `|` isn't necessary. 

Also can you prefix this with `([ -z $plist_secret ] || scripts/third_party/travis/retry.sh` similar to the other quickstarts? The first is a check to see if the secret is `null` in which case we'll fail early since the right info isn't available, and second will retry any failures in case it's a flake.